lib/screens/auth/auth_screen.dart

```diff
@@ -0,0 +1,157 @@
+import 'package:firebase_auth/firebase_auth.dart';
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+import 'package:mung_ge_mung_ge/providers/auth_provider.dart';
+import 'package:mung_ge_mung_ge/models/logInData.dart';
+
+class AuthScreen extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return ChangeNotifierProvider(
+      create: (BuildContext context) => LogInController(auth: FirebaseAuth.instance),
+      child: AuthScreenContent(),
+    );
+  }
+}
+
+class AuthScreenContent extends StatefulWidget {
+
+  @override
+  _AuthScreenContentState createState() => _AuthScreenContentState();
+}
+
+class _AuthScreenContentState extends State<AuthScreenContent> {
+  final GlobalKey<FormState> _key = new GlobalKey();
+  bool _validate = false;
+  final LoginRequestData _loginData = LoginRequestData();
+  bool _obscureText = true;
+
+  @override
+  Widget build(BuildContext context) {
+    LogInController _logInModel = Provider.of<LogInController>(context);
+
+    return new Scaffold(
+      body: new Center(
+        child: new SingleChildScrollView(
+          child: new Container(
+            margin: new EdgeInsets.all(20.0),
+            child: Center(
+              child: new Form(
+                key: _key,
+                child: _getFormUI(_logInModel),
+              ),
+            ),
+          ),
+        ),
+      ),
+    );
+  }
+
+  Widget _getFormUI(LogInController _logInModel) {
+    return new Column(
+      children: <Widget>[
+        Icon(
+          Icons.cloud_circle_outlined,
+          color: Colors.lightBlue,
+          size: 150.0,
+        ),
+        new SizedBox(height: 50.0),
+        new TextFormField(
+          keyboardType: TextInputType.emailAddress,
+          autofocus: false,
+          decoration: InputDecoration(
+            hintText: 'Email',
+            contentPadding: EdgeInsets.fromLTRB(20.0, 15.0, 20.0, 15.0),
+            border:
+            OutlineInputBorder(borderRadius: BorderRadius.circular(32.0)),
+          ),
+          //validator: FormValidator().validateEmail,
+          onSaved: (String? value) {
+            _loginData.email = value!;
+          },
+        ),
+        new SizedBox(height: 20.0),
+        new TextFormField(
+            autofocus: false,
+            obscureText: _obscureText,
+            keyboardType: TextInputType.text,
+            decoration: InputDecoration(
+              hintText: 'Password',
+              contentPadding: EdgeInsets.fromLTRB(20.0, 10.0, 20.0, 10.0),
```

Comment:
```suggestion
              contentPadding: EdgeInsets.symmetric(horizontal: 20.0, vertical: 10.0),
```
저는 개인적으로 상하, 좌우 padding 값이 같다면, 이렇게 쓰는 게 조금 더 가독성 있는 것 같습니다!
개인적인 의견이니 무시하셔도 됩니당 ☺️